### PR TITLE
Miscellaneous grammar fixes, wording updates

### PR DESCRIPTION
**Pull request by jonathanwuki**
Found a few places with incorrect grammar, also updated the wording in some places in an attempt to make things slightly more clear.

### CODE REVIEW

**Comment by khronokernel** [comment]
Thanks for the PR! Unfortunately this commit will likely conflict with tomorrow's 0.6.2 release so we'll need to hold off on review until that's merged to see what needs adjusting. 

Also should mention we have a build system in place that verifies spelling, this is covered under our [CONTRIBUTING.md](https://dortania.github.io/OpenCore-Install-Guide/CONTRIBUTING.html) file. You can check yourself by clicking the red x next to the commit, it'll send you to the logs of Build, Lint and spellcheck. From there you'll be directed to travis:

* https://travis-ci.com/github/dortania/OpenCore-Install-Guide/jobs/394696099

As you can see, `x86-based` and `x64` were set off. So they need to be added to the [Dictionary.txt](https://github.com/dortania/OpenCore-Install-Guide/blob/master/dictionary/dictionary.txt) file, then run `npm run sort-dict` to sort the entries. Annoyingly `Installing macOS on an x86-based PC` set off the spellcheck as well for some reason, and changing the rules have issues with breaking other parts of spellcheck. So I recommend either reverting that change or try capitalizing the X.

But besides that, a quick skim shows it's overall good for merge. Thanks again!

**Comment by jonathanwuki** [comment]
Fixed – totally missed that when I read the CONTRIBUTING.md file – thanks for the info!

**Review by khronokernel — APPROVED** [review]

**Comment by khronokernel** [comment]
Thank you again for the PR, I seriously appreciate it!

**Pull request merged**
